components/admin/requests/payment-information/Card.tsx

```diff
@@ -133,33 +141,64 @@ const Card: FC<Props> = props => {
             Fees
           </Text>
         </Box>
-        <SimpleGrid columns={2} spacingX="70px" spacingY="12px">
-          <Box>
-            <Text as="p" textStyle="body-regular" textAlign="left">
-              Permit Fee
-            </Text>
-          </Box>
-          <Box>
-            <Text as="p" textStyle="body-regular" textAlign="left">
-              ${processingFee}
-            </Text>
-          </Box>
-          <Box>
-            <Text as="p" textStyle="body-regular" textAlign="left">
-              Donation
-            </Text>
-          </Box>
-          <Box>
-            <Text as="p" textStyle="body-regular" textAlign="left">
-              ${donationAmount}
-            </Text>
-          </Box>
-          <Box>
-            <Text as="p" textStyle="caption" textAlign="left">
-              Paid with {titlecase(paymentMethod)}
-            </Text>
-          </Box>
-        </SimpleGrid>
+        <HStack>
+          <SimpleGrid columns={2} spacingX="70px" spacingY="12px">
+            <Box>
+              <Text as="p" textStyle="body-regular" textAlign="left">
+                Permit Fee
+              </Text>
+            </Box>
+            <Box>
+              <Text as="p" textStyle="body-regular" textAlign="left">
+                ${processingFee}
+              </Text>
+            </Box>
+            <Box>
+              <Text as="p" textStyle="body-regular" textAlign="left">
+                Donation
+              </Text>
+            </Box>
+            <Box>
+              <Text as="p" textStyle="body-regular" textAlign="left">
+                ${donationAmount}
+              </Text>
+            </Box>
+            <Box>
+              <Text as="p" textStyle="caption" textAlign="left">
+                Paid with {titlecase(paymentMethod)}
+              </Text>
+            </Box>
+          </SimpleGrid>
+          {hasSecondPaymentMethod && (
```

Comment:
Similar question: why not check if paymentMethod2 is null instead of using a special flag?

Reply:
See [comment](https://github.com/uwblueprint/richmond-centre-for-disability/pull/338#discussion_r1485737141)